### ISSUE TITLE
unit buffs stack

### ISSUE THREAD

**Issue by fsolecki** [Blizzard/s2client-api#151]
Is it normal that Unit buffs stack in `unit->buffs` (not removed between each step) ? 
In comparison to orders that are emptied.

In `sc2_proto_to_pods.cc:bool Convert(const ObservationRawPtr& observation_raw, UnitPool& unit_pool, uint32_t game_loop)`:
`unit->buffs.clear(); `
seems to be missing before
```
for (int buff_index = 0; buff_index < observation_unit.buff_ids_size(); ++buff_index) {
   unit->buffs.push_back(observation_unit.buff_ids(buff_index));
}
```

**Comment by AnthonyBrunasso** [comment]
Ah, this appears to be a an oversight from the unit pointer refactor.

I will fix this and link to this issue in the commit. Thank you!

**Comment by AnthonyBrunasso** [comment]
Resolved by #152
